packages/financial-templates-lib/src/price-feed/DefaultPriceFeedConfigs.js

```diff
@@ -554,7 +554,7 @@ const defaultConfigs = {
         uniswapAddress: "0xcd7989894bc033581532d2cd88da5db0a4b12859",
         invertPrice: true
       },
-      BADGER_USD_HUOBI: { type: "cryptowatch", exchange: "huobi", pair: "badgerusdt" },
+      BADGER_USD_HUOBI: { type: "cryptowatch", exchange: "huobi", pair: "badgerusdt", twapLength: 0 },
```

Comment:
Doesn't this default to 0?

Reply:
Yeah, but we define `twapLength` a level above for the uniswap feeds, so that gets inherited by this config. To set it back to 0, it needs to be specified.